src/sql/connection.py

```diff
@@ -78,7 +78,7 @@ def get_missing_package_suggestion_str(e):
         module_name, MISSING_PACKAGE_LIST_EXCEPT_MATCHERS.keys()
     )
     if close_matches:
-        return "Perhaps you meant to use driver the dialect: \"{}\"".format(
+        return 'Perhaps you meant to use driver the dialect: "{}"'.format(
```

Comment:
cleaner version:

```
f"this is the value of something: {something!r}"
```

this ensures the quotes are added when printing the value in "something"

Reply:
Fix in [Add: f-string fix](https://github.com/ploomber/jupysql/pull/171/commits/993e9a11c98ac198440ca4b8d0cc1438617fdf51)

Still need to keep the \" since we need to display double quote, Ref: [Quotations](https://www.datacamp.com/tutorial/f-string-formatting-in-python#4.1.-quotations)

If uses `{something!r}`, it will present single quote which we don't want